### PR TITLE
Adding Adverline to amp-ad

### PR DESCRIPTION
**Pull request by benjaminclot**
Adding [Adverline](http://www.adverline.com) AMP support

### CODE REVIEW

**Comment by benjaminclot** [comment]
@googlebot The CLA has been signed as Adverline

File: examples/ads.amp.html [hunk 1]
```diff
@@ -287,6 +288,15 @@ <h2 id="aduptech">Ad Up Technology</h2>
     <div fallback></div>
   </amp-ad>
 
+  <h2 id="adverline">Adverline</h2>
+  <amp-ad width=300 height=250
+          type="adverline"
```
Comment (lannka) [inline examples/ads.amp.html]: please indent 4 spaces.
could you also fix the following example for AdvertServce?

**Comment by lannka** [comment]
@benjaminclot i don't see Adverline in our signed CLA corp list. Can you double confirm if it's already done or still in process?

**Comment by lannka** [comment]
Please also fix the failing test.

**Comment by benjaminclot** [comment]
@lannka The CLA has been signed, I can provide you with a link (but maybe not publicly?...)

**Comment by benjaminclot** [comment]
@lannka the remaining failing test also happens on the current master branch, I don't think it is related to this PR.

**Comment by lannka** [comment]
@benjaminclot the remaining tests are not your fault.

Is "Advertline" the exact corp name you used when signing the CLA?

**Comment by benjaminclot** [comment]
@lannka It is Adverline (without a "t") and yes it is the exact corp name

**Comment by benjaminclot** [comment]
@lannka I just received the CLA countersigned by Google, you should be able to view it now

**Comment by lannka** [comment]
awesome, merging

**Comment by benjaminclot** [comment]
Just a quick follow-up: when will it be available in production?

**Pull request merged**